src/platforms/web/server.js

```diff
@@ -1,12 +1,61 @@
+const { readFileSync, writeFileSync } = require('fs');
 const express = require('express');
+const fetch = require('node-fetch');
 const path = require('path');
 const app = express();
 const port = 1337;
+const headRegex = /(<head>).*(<\/head>)/g;
 
 app.use(express.static(__dirname));
 
-app.get('*', function(req, res) {
-  res.sendFile(path.join(__dirname, '/index.html'));
+app.get('*', async(req, res) => {
+  if (req.url.includes('/$/')) {
+    res.sendFile(path.join(__dirname, '/index.html'));
+  }
+
+  const claimId = req.url.split('/').pop();
+  const queryUrl = `https://chainquery.lbry.io/api/sql?query=SELECT * FROM claim where claim_id="${claimId}"`;
```

Comment:
Query with a channel: SELECT channel.name as channel, claim.name, claim.description, claim.language, claim.thumbnail_url, claim.title FROM claim left join claim channel on claim.publisher_id = channel.claim_id where claim.claim_id="93a61971c473ade02300361dc077eb564b558a8d"

Will we always have the claim id? does that come from the claim that's resolved? What if someone shares beta.lbry.tv/jiggytom Sounds like we'll need to use claim.name = xxx and bid_state = controlling per my original suggestion in that case.

Reply:
I saw we grab claim id from the URL, so we'll need to adjust the query as such.